docs/src/example/drawerNavItem/BasicDrawerNavItemExample.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { Drawer, DrawerBody, DrawerNavGroup, DrawerNavItem } from '@brightlayer-ui/react-components';
+import * as colors from '@brightlayer-ui/colors';
+
+export const BasicDrawerNavItemExample = (): JSX.Element => {
```

Comment:
What's the purpose of having this? Should this component exist or should this all be contained within the `BasicDrawerNavItem`?

Reply:
The purpose of this file is, when user clicks on Full Code on GH link, it should navigate to this code, we don't want to show user the other styling, wrapper/conatainer part.